src/dialog/tooltip/tooltip.directive.spec.ts

```diff
@@ -44,131 +49,87 @@ describe("Tooltip directive", () => {
-		button.click();
-		fixture.detectChanges();
+		expect(button.nativeElement.getAttribute("placement")).toBe("top");
 
 		expect(fixture.componentInstance instanceof TooltipTest).toBe(true);
-		expect(document.querySelector(".tooltip--bottom-left")).not.toBe(null);
 	});
 
-	xit("should create the tooltip component and tooltip should appear at the bottom right", () => {
-		TestBed.overrideComponent(TooltipTest, {
-			set: {
-				template: "<button ibmTooltip='Hello There' placement='bottom-right'>Me</button>"
-			}
-		});
+	it("should expand tooltip on click and emit an onOpen event", () => {
 		const fixture = TestBed.createComponent(TooltipTest);
+		let wrapper = fixture.componentInstance;
+		spyOn(wrapper, "onOpen");
 		fixture.detectChanges();
 
-		let button = fixture.nativeElement.querySelector("button");
+		let button = fixture.debugElement.query(By.css("button"));
 
-		button.click();
-		fixture.detectChanges();
-
-		expect(fixture.componentInstance instanceof TooltipTest).toBe(true);
-		expect(document.querySelector(".tooltip--bottom-right")).not.toBe(null);
-	});
-
-	xit("should create the tooltip component and tooltip should appear at the left", () => {
-		TestBed.overrideComponent(TooltipTest, {
-			set: {
-				template: "<button ibmTooltip='Hello There' placement='left'>Me</button>"
-			}
-		});
-		const fixture = TestBed.createComponent(TooltipTest);
-		fixture.detectChanges();
-
-		let button = fixture.nativeElement.querySelector("button");
+		button.nativeElement.click();
 
-		button.click();
 		fixture.detectChanges();
 
-		expect(fixture.componentInstance instanceof TooltipTest).toBe(true);
-		expect(document.querySelector(".tooltip--left")).not.toBe(null);
+		expect(button.nativeElement.getAttribute("aria-expanded")).toBe("true");
+		expect(wrapper.onOpen).toHaveBeenCalled();
 	});
 
-	xit("should create the tooltip component and tooltip should appear at the right", () => {
-		TestBed.overrideComponent(TooltipTest, {
+	it("should set tooltip type to warning", () => {
+		const fixture = TestBed.overrideComponent(TooltipTest, {
 			set: {
-				template: "<button ibmTooltip='Hello There' placement='right'>Me</button>"
+				template: `
+					<button ibmTooltip="Hello There" tooltipType="warning">Me</button>
+					<ibm-placeholder></ibm-placeholder>
+				`
 			}
-		});
-		const fixture = TestBed.createComponent(TooltipTest);
-		fixture.detectChanges();
+		}).createComponent(TooltipTest);
 
-		let button = fixture.nativeElement.querySelector("button");
-
-		button.click();
 		fixture.detectChanges();
 
-		expect(fixture.componentInstance instanceof TooltipTest).toBe(true);
-		expect(document.querySelector(".tooltip--right")).not.toBe(null);
+		let button = fixture.debugElement.query(By.css("button"));
+
+		expect(button.nativeElement.getAttribute("tooltiptype")).toBe("warning");
 	});
 
-	xit("tooltip should appear auto as default", () => {
-		TestBed.overrideComponent(TooltipTest, {
+	it("should set tooltip type to error", () => {
```

Comment:
`tooltipType` doesn't seem to actually do anything? Maybe we can drop this test?